module/Company/view/company/company/jobs.phtml

```diff
@@ -62,6 +62,13 @@ $this->headTitle($category->getPluralName());
                             <p class="job-email">
                                 <a href="mailto:<?= rawurlencode($job->getEmail()) ?>"><?= $job->getEmail() ?></a>
                             </p>
+                            <?php if ($job->getWebsite() !== ""): ?>
+                                <p class="job-website">
+                                    <a href="<?= $escaper->escapeHtmlAttr($job->getWebsite()) ?>" rel="noreferrer">
```

Comment:
I don't think escape HTML is neccesary here, but it's not blocking.

Reply:
@Koen1999 Yes, escaping is necessary, otherwise stored XSS is possible. I know this is highly unlikely, but I do not want the website to break because some company has a funny URL or one of the admins goes rogue.